app/thirdparty/java/other/writeily/widget/WrMarkorWidgetProvider.java

```diff
@@ -121,14 +119,4 @@ public void onUpdate(Context context, AppWidgetManager appWidgetManager, int[] a
         }
         super.onUpdate(context, appWidgetManager, appWidgetIds);
     }
-
-    public static void handleWidgetScheme(Context context, RemoteViews remoteViews, Boolean enabled) {
-        if (enabled) {
```

Comment:
@gsantner I have updated the widget scheme so that it follows the _system_ theme instead of the app theme. This simplifies a lot of code and makes the widget break less. If this is not acceptable, we can revert this change.

Reply:
Thats good